### PR TITLE
Add missing test for clearing up local cache on invalid parameters error.

### PR DESCRIPTION
**Pull request by mechanicles**
Add missing test for clearing up local cache and check response should
be present on invalid parameters error.

?@rafaelfranca

### CODE REVIEW

**Comment by mechanicles** [comment]
Sorry as @rafaelfranca had worked on similar stuff so assigning him.

r? @rafaelfranca

**Comment by mechanicles** [comment]
@rafaelfranca codeclimate check is passed actually but not reflecting here. Please review when you get the time.

File: activesupport/test/caching_test.rb [hunk 1]
```diff
@@ -25,6 +25,18 @@ def test_local_cache_cleared_on_close
             assert_nil LocalCacheRegistry.cache_for(key)
           end
 
+          def test_local_cache_cleared_and_response_shouold_be_present_on_invalid_parameters_error
```
Comment (rafaelfranca) [inline activesupport/test/caching_test.rb]: `should`

**Comment by mechanicles** [comment]
@rafaelfranca Updated. Please check.

File: activesupport/test/caching_test.rb [hunk 2]
```diff
@@ -25,6 +25,18 @@ def test_local_cache_cleared_on_close
...
 
+          def test_local_cache_cleared_and_response_should_be_present_on_invalid_parameters_error
+            key = "super awesome key"
+            assert_nil LocalCacheRegistry.cache_for key
+            middleware = Middleware.new('<3', key).new(->(env) {
+              assert LocalCacheRegistry.cache_for(key), 'should have a cache'
+              raise Rack::Utils::InvalidParameterError
+            })
+            response = middleware.call({})
+            assert response, 'should be exist'
```
Comment (prathamesh-sonpatki) [inline activesupport/test/caching_test.rb]: Can you change the error message?

Comment (mechanicles) [inline activesupport/test/caching_test.rb]: @prathamesh-sonpatki Error message? What do you expect here?

Comment (prathamesh-sonpatki) [inline activesupport/test/caching_test.rb]: We can change `should be exist` to something like `response should exist`

**Comment by mechanicles** [comment]
@prathamesh-sonpatki Updated. Thanks.

**Pull request merged**